### DISCUSSION TITLE
如何实时增加网络中的节点

### DISCUSSION THREAD

**Discussion by qq9077** [opengnb/opengnb#37] [Ideas]
例如我使用他组成了一个网络玩着游戏的时候，我新的人中途加入，在不断开现有网络的情况下如何让gnb更新新的节点。
之前的游戏可以中途关闭并重新打开会保留原有进度。
现在遇到不能中途关闭的游戏了，是否能提供一个功能或者一个全新的控制台程序，
使得我们能使用 命令增加新的节点nodeid，ip，和publickey，这些将自动更新到conf里的文件中并更新 到ctrl-block中。

**Comment by qq9077** [comment]
我给opengnb制作了一个低门槛使用的控制台
http://14.29.224.25:8793/gnb/index

**Reply by gnbdev** [reply]
1、opengnb 是一个开源项目，主要目的是分享代码倡导用户按自己的想法修改代码，重在参与，虽然主线代码不轻易接受PR。

2、opengnb的安全策略非常保守，未来在考虑清楚各种细节后，在线更新配置增减节点的功能可能会有，opengnb 的底层代码架构支持在线热更新配置。

**Comment by gnbdev** [comment]
> 我给opengnb制作了一个低门槛使用的控制台 http://14.29.224.25:8793/gnb/index

方便留个微信，邀请进技术支持群，或许群里的朋友需要
